src/utils/miscellaneous.ts

```diff
@@ -88,6 +88,10 @@ export function isTradeFilled(trade: Trade): boolean {
   return isAmountDifferenceGreaterThanNegligibleAmount(trade.sellAmount, trade.remainingAmount)
 }
 
+export function isTradeSettled(trade: Trade): boolean {
+  return trade.settlingDate <= new Date()
```

Comment:
not that it matters, but why did you choose to deal with Date as object rather than timestamp as number.
Like here `trade.settlingDate <= Date.now()` and in Trade type
It ultimately doesn't matter, but ? remains

Reply:
Didn't really think about it. Will revise for consistency.